games_registry.py

```
@@ -0,0 +1,841 @@
+    slug = re.sub(r"[^a-zA-Z0-9]+", "_", normalized).strip("_").lower()
+    if slug:
+        return slug
+    fallback = re.sub(r"[^a-zA-Z0-9]+", "_", name).strip("_").lower()
+    return fallback or "project"
+
+
+def normalize_translation_status(raw: str) -> str:
+    text = raw.strip()
+    if not text:
+        return "待确认"
+    base = re.sub(r"（[^）]*）", "", text).strip()
```

Ruff (0.15.20)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)